packs/data/pathfinder-bestiary-3.db/werecrocodile.json

```
@@ -635,15 +646,15 @@
                     "value": null
                 },
                 "description": {
-                    "value": "<p>This curse affects only humanoids</p>\n<p><strong>Saving Throw</strong> @Check[type:fortitude|dc:15] DC.</p>\n<p>On each full moon, the cursed creature must succeed at another Fortitude save or turn into a werecrocodile until dawn. The creature is under the GM's control and goes on a rampage for half the night before falling unconscious until dawn.</p>"
+                    "value": "<p>This curse affects only humanoids.</p>\n<hr />\n<p><strong>Saving Throw</strong> @Check[type:fortitude|dc:15|name:Curse of the Werecrocodile]</p>\n<p>On each full moon, the cursed creature must succeed at another Fortitude save or turn into a werecrocodile until dawn.</p>\n<p>The creature is under the GM's control and goes on a rampage for half the night before falling unconscious until dawn.</p>"
```

```suggestion
                    "value": "<p>This curse affects only humanoids.</p>\n<hr />\n<p><strong>Saving Throw</strong> @Check[type:fortitude|dc:15]</p>\n<p>On each full moon, the cursed creature must succeed at another Fortitude save or turn into a werecrocodile until dawn. The creature is under the GM's control and goes on a rampage for half the night before falling unconscious until dawn.</p>"
```
I think we may need to clean up the family abilities for werecreatures.